tendrl/gluster_integration/objects/definition/gluster.yaml

```diff
@@ -105,7 +78,7 @@ namespace.gluster:
           help: raw data
           type: String
       enabled: true
-      value: clusters/$TendrlContext.integration_id/raw_map/
+      value: clusters/$TendrlContext.integration_id/nodes/$NodeContext.node_id/raw_map/
```

Comment:
If I understand correctly this for maintaining objects's raw representation. right? If so why under node specific dir?
Can you explain the logic a bit?

Reply:
Yes. raw map changes per node. Hence saving pee node

Reply:
ok